package.json

```diff
@@ -40,5 +40,8 @@
         "prettier": "^2.2.1",
         "prettier-plugin-apex": "^1.8.0",
         "semver": "^7.3.4"
+    },
+    "dependencies": {
```

Comment:
@codefriar was this intentional? I defer this to @pozil as I am not sure if this is needed!

Reply:
Thanks for removing it @codefriar. Is the `package-lock.json` up to date with that change?